server/src/internalClusterTest/java/org/opensearch/search/SearchWeightedRoutingIT.java

```diff
@@ -615,7 +615,7 @@ public void testSearchAggregationWithNetworkDisruption_FailOpenEnabled() throws
         for (int i = 0; i < 50; i++) {
             responses[i] = internalCluster().client(nodeMap.get("b").get(0))
                 .prepareSearch("index")
-                .setSize(10)
+                .setSize(20)
```

Comment:
wondering how it fixed the test .

Reply:
This was not the only change. Made some changes here as well  https://github.com/opensearch-project/OpenSearch/pull/5072/commits/e76debab289ac087be13f1d25f018a662dc0be1f. 
Basically the search requests were hitting node with network disruption earlier which was causing shard failures